.appveyor.yml

```
@@ -95,6 +95,29 @@ environment:
         - TOXENV: py37-piplatest-coverage
           PIP: latest
 
+        - TOXENV: py38-pip8.1.1
+          PIP: 8.1.1
+        - TOXENV: py38-pip9.0.1
+          PIP: 9.0.1
+        - TOXENV: py38-pip9.0.3
+          PIP: 9.0.3
+        - TOXENV: py38-pip10.0.1
+          PIP: 10.0.1
+        - TOXENV: py38-pip18.0
+          PIP: 18.0
+        - TOXENV: py38-pip19.0.3
+          PIP: 19.0.3
+        - TOXENV: py38-pip19.1-coverage
+          PIP: 19.1
+        - TOXENV: py38-pip19.2.3
+          PIP: 19.2.3
+        - TOXENV: py38-pip19.3
```

```suggestion
        - TOXENV: py38-pip19.3-coverage
```
Add coverage for pip19.3